### PR TITLE
add fast tune doc

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
add fast tune doc

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/10487
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: grafana-tikv-dashboard.md [hunk 1]
```diff
@@ -424,3 +426,15 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
     - raw_delete：删除一个 key/value 对
     - raw_batch_delete：删除一批 key/value 对
     - raw_delete_range：删除连续的一段区间
+
+## TiKV-FastTune 面板
+
+如果 TiKV 出现 QPS 抖动、延迟抖动、延迟增加趋势等性能问题，你可以查看 **TiKV-FastTune** 面板。**TiKV-FastTune** 包括多组面板，可帮助你诊断性能问题，尤其适用于集群中写入负载较大的场景。
```
Comment (innerr) [inline grafana-tikv-dashboard.md:432]: 如果 => 当 ... 时

Comment (innerr) [reply grafana-tikv-dashboard.md:432]: 查看 ... 页面
这里两个面板含义不同

File: grafana-tikv-dashboard.md [hunk 2]
```diff
@@ -424,3 +426,15 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
...
+
+当出现写入相关的性能问题时，可以先查看 TiDB 面板，以及 Grafana 中 **TiDB Summary** 面板下的 **TiKV latency** 面板。如果问题出在存储端，打开 **TiKV-FastTune** 页面，浏览并检查上面的每个面板。
```
Comment (innerr) [inline grafana-tikv-dashboard.md:434]: 面板 => 页面

File: grafana-tikv-dashboard.md (hunk 1 above)
Comment (TomShawn) [reply grafana-tikv-dashboard.md:432]:
```suggestion
当 TiKV 出现 QPS 抖动、延迟抖动、延迟增加趋势等性能问题时，你可以查看 **TiKV-FastTune** 页面。**TiKV-FastTune** 包括多组面板，可帮助你诊断性能问题，尤其适用于集群中写入负载较大的场景。
```

File: grafana-tikv-dashboard.md (hunk 2 above)
Comment (TomShawn) [reply grafana-tikv-dashboard.md:434]:
```suggestion
当出现写入相关的性能问题时，可以先查看 TiDB 页面，以及 Grafana 中 **TiDB Summary** 页面下的 **TiKV latency** 面板。如果问题出在存储端，打开 **TiKV-FastTune** 页面，浏览并检查上面的每个面板。
```

File: grafana-tikv-dashboard.md [hunk 3]
```diff
@@ -424,3 +426,15 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
...
+
+在 **TiKV-FastTune** 页面中，标题描述了性能问题的可能成因。要验证成因是否正确，你需要检查页面上的图表。
```
[the remark this answers is not in the document]
Comment (TomShawn) [reply grafana-tikv-dashboard.md:436]:
```suggestion
在 **TiKV-FastTune** 每个面板中，标题描述了性能问题的可能成因。要验证成因是否正确，你需要检查左右两组页面上的图表曲线。
```

File: grafana-tikv-dashboard.md [hunk 4]
```diff
@@ -424,3 +426,15 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
...
+
+左图 Y 轴表示存储端的 write-RPC QPS，右图 Y 轴上的一组图是倒置绘制的。如果左图的形状与右图的形状匹配，则标题描述的问题成因是正确的。
```
[the remark this answers is not in the document]
Comment (TomShawn) [reply grafana-tikv-dashboard.md:438]:
```suggestion
左边 Y 轴表示存储端的 write-RPC QPS，右边 Y 轴上的一组图是倒置绘制的。如果左边 Y 轴的曲线形状与右边的形状匹配，则面板标题描述的问题成因是正确的。
```

File: grafana-tikv-dashboard.md [hunk 5]
```diff
@@ -424,3 +426,15 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
...
+
+当 TiKV 出现 QPS 抖动、延迟抖动、延迟增加趋势等性能问题时，你可以查看 **TiKV-FastTune** 页面。**TiKV-FastTune** 包括多组面板，可帮助你诊断性能问题，尤其适用于集群中写入负载较大的场景。
+
+当出现写入相关的性能问题时，可以先查看 TiDB 页面，以及 Grafana 中 **TiDB Summary** 页面下的 **TiKV latency** 面板。如果问题出在存储端，打开 **TiKV-FastTune** 页面，浏览并检查上面的每个面板。
+
+在 **TiKV-FastTune** 每个面板中，标题描述了性能问题的可能成因。要验证成因是否正确，你需要检查左右两组页面上的图表曲线。
+
+左边 Y 轴表示存储端的 write-RPC QPS，右边 Y 轴上的一组图是倒置绘制的。如果左边 Y 轴的曲线形状与右边的形状匹配，则面板标题描述的问题成因是正确的。
+
+有关该面板的监控项以及解释，参考 [TiKV-FastTune 用户手册（英文）](https://docs.google.com/presentation/d/1aeBF2VCKf7eo4-3TMyP7oPzFWIih6UBA53UI8YQASCQ/edit#slide=id.gab6b984c2a_1_352)。
```
Comment (lilin90) [inline grafana-tikv-dashboard.md:440]:
```suggestion
当 TiKV 出现 QPS 抖动、延迟抖动、延迟增加趋势等性能问题时，你可以查看 **TiKV-FastTune** 面板。**TiKV-FastTune** 包括多组子面板，可帮助你诊断性能问题，尤其适用于集群中写入负载较大的场景。

当出现写入相关的性能问题时，可以先在 Grafana 中查看 TiDB 面板，**TiDB Summary** 面板下的 **TiKV latency** 指标。如果问题出在存储端，打开 **TiKV-FastTune** 面板，浏览并检查上面的每个指标。

在 **TiKV-FastTune** 的面板中，指标标题描述了性能问题的可能成因。要验证成因是否正确，你需要检查具体的图表曲线。

左边 Y 轴表示存储端的 write-RPC QPS，右边 Y 轴上的一组图是倒置绘制的。如果左边 Y 轴的曲线形状与右边的形状匹配，则指标标题描述的问题成因是正确的。

有关该面板的具体监控项以及解释，参考 [TiKV-FastTune 用户手册（英文）](https://docs.google.com/presentation/d/1aeBF2VCKf7eo4-3TMyP7oPzFWIih6UBA53UI8YQASCQ/edit#slide=id.gab6b984c2a_1_352)。
```

File: grafana-tikv-dashboard.md [hunk 6]
```diff
@@ -424,3 +426,15 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
     - raw_delete：删除一个 key/value 对
     - raw_batch_delete：删除一批 key/value 对
     - raw_delete_range：删除连续的一段区间
+
+## TiKV-FastTune 面板
+
+当 TiKV 出现 QPS 抖动、延迟抖动、延迟增加趋势等性能问题时，你可以查看 **TiKV-FastTune** 面板。**TiKV-FastTune** 包括多组子面板，可帮助你诊断性能问题，尤其适用于集群中写入负载较大的场景。
+
+当出现写入相关的性能问题时，可以先在 Grafana 中查看 TiDB 面板，**TiDB Summary** 面板下的 **TiKV latency** 指标。如果问题出在存储端，打开 **TiKV-FastTune** 面板，浏览并检查上面的每个指标。
```
Comment (TomShawn) [inline grafana-tikv-dashboard.md:434]:
```suggestion
当出现写入相关的性能问题时，可以先在 Grafana 中查看 TiDB 面板。如果问题出在存储端，打开 **TiKV-FastTune** 面板，浏览并检查上面的每个指标。
```

File: grafana-tikv-dashboard.md [hunk 7]
```diff
@@ -424,3 +426,15 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
     - raw_delete：删除一个 key/value 对
     - raw_batch_delete：删除一批 key/value 对
     - raw_delete_range：删除连续的一段区间
+
+## TiKV-FastTune 面板
+
+当 TiKV 出现 QPS 抖动、延迟抖动、延迟增加趋势等性能问题时，你可以查看 **TiKV-FastTune** 面板。**TiKV-FastTune** 包括多组子面板，可帮助你诊断性能问题，尤其适用于集群中写入负载较大的场景。
+
+当出现写入相关的性能问题时，可以先在 Grafana 中查看 TiDB 面板。如果问题出在存储端，打开 **TiKV-FastTune** 面板，浏览并检查上面的每个指标。
```
Comment (TomShawn) [inline grafana-tikv-dashboard.md:434]:
```suggestion
当出现写入相关的性能问题时，可以先在 Grafana 中查看 TiDB 相关的面板。如果问题出在存储端，打开 **TiKV-FastTune** 面板，浏览并检查上面的每个指标。
```

**Pull request merged**